### ISSUE TITLE
Not allowed to reboot system like normal after update

### ISSUE THREAD

**Issue by ty1118** [dnschneid/crouton#4539]

**Comment by MrHallway0** [comment]
Do you have any of the following that you could give? 
 - Logs
 - Version numbers
 - Detailed description 
 - System information

**Comment by strawwmann** [comment]
Sounds like the same issue I had when updating a Lenovo IdeaPad Flex 3 to Chrome OS v91.0.4472.165 (64bit) a few days ago (on a machine with Crouton installed where all had been working well before - Crouton was installed only a few weeks ago using the current version with manual fixes for the CRAS audio problem:, as per Issues #4506 #4507).

After allowing the Chrome OS update to download a notification indicated that the system needed to reboot to complete installation, which I accepted.
When I tried 'Ctrl-D' in the boot menu, the machine froze on a white screen with the Chrome logo.
After 10+ min pauses, I tried restarting several times (using power button) with the same result.
I then left it overnight on the frozen screen, but it did not resolve.

I eventually had to take the 'Space key' boot menu option to revert the machine to factory settings (then set up Chrome OS from scratch, back in non-developer mode).
(I haven't been game to reinstall Crouton yet - I'm waiting to see how wide-spread the problem is, and whether a fresh install of Crouton will work).

There are no logs to report - everything was wiped in the factory reset.
